### PR TITLE
docs: add blog post for js-ipfs 0.54.x

### PR DESCRIPTION
**Pull request by achingbrain**
Also removes the welcome bot from the contributor thanks, I'm not sure the robot needs thanking..

### CODE REVIEW

File: content/post/120-js-ipfs-0.54.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+title: js-IPFS 0.54.0 brings UPnP NAT Hole Punching to your js-IPFS node
+header_image: js-ipfs-placeholder.png
+author: Alex Potsides
+---
+
+# 🔦 Highlights
+
+> NAT UPnP Hole Punching
+
+`js-IPFS@0.53.0` has launched into the stratosphere with an experimental [gRPC][]-over-websockets server that means the http client has true full-duplex streaming for the `ipfs.add` API
```
Comment (vasco-santos) [inline content/post/120-js-ipfs-0.54.md:13]: This seems from the previous post
```suggestion
```

**Review by vasco-santos — CHANGES_REQUESTED** [review]
Except for my change suggestion, LGTM

File: content/post/120-js-ipfs-0.54.md (hunk 1 above)
Comment (achingbrain) [reply content/post/120-js-ipfs-0.54.md:13]: Oops, me copy-pasta

Comment (achingbrain) [reply content/post/120-js-ipfs-0.54.md]: I've re-written it

**Review by jennwrites — APPROVED** [review]
Thanks for the backlog cleanup!!

**Review by vasco-santos — APPROVED** [review]

**Pull request merged**